kuryr_kubernetes/controller/managers/prometheus_exporter.py

```diff
@@ -134,3 +142,12 @@ def _create_metrics(self):
         self.pod_creation_latency = prometheus_client.Histogram(
             'kuryr_pod_creation_latency', 'Time taken for a pod to have'
             ' Kuryr annotations set', buckets=buckets, registry=self.registry)
+
+        self.load_balancer_readiness = prometheus_client.Count(
+            'kuryr_load_balancer_readiness',
+            'Increasing this metric indicates issues with creating Octavia LB',
```

Comment:
How about : "This metric is increased when Kuryr notices that an Octavia load balancer is stuck in an unexpected state"?